src/ensembl/src/content/app/browser/browser-nav/BrowserNavBarControls.test.tsx

```diff
@@ -0,0 +1,56 @@
+import React from 'react';
+import { mount } from 'enzyme';
+import faker from 'faker';
+import times from 'lodash/times';
+
+import BrowserNavIcon from './BrowserNavIcon';
+
+import { BrowserNavBarControls } from './BrowserNavBarControls';
+import { BrowserNavStates } from '../browserState';
+
+jest.mock('./BrowserNavIcon', () => () => <div>BrowserNavIcon</div>);
+
+const browserNavStates = times(6, faker.random.boolean) as BrowserNavStates;
+
+describe('BrowserNavBarControls', () => {
+  it('renders without errors', () => {
+    expect(() =>
+      mount(
+        <BrowserNavBarControls
+          browserNavStates={browserNavStates}
+          disabled={faker.random.boolean()}
+        />
+      )
+    ).not.toThrow();
+  });
+
+  it('disables all buttons if the disabled prop is set to true', () => {
+    const wrapper = mount(
+      <BrowserNavBarControls
+        browserNavStates={browserNavStates}
+        disabled={true}
+      />
+    );
+    const controlButtons = wrapper.find(BrowserNavIcon);
+    expect(controlButtons.length).toEqual(browserNavStates.length);
+    controlButtons.forEach((button) => {
+      expect(button.props().enabled).toBe(false);
```

Comment:
Wouldn't it be better to just get the only prop we want like this:
`button.prop('enabled')`?

Reply:
✔️